### PR TITLE
update coverage dependency

### PR DESCRIPTION
**Pull request by oliviadawd**
updating `coverage` dependency for big sur. here's the error im trying to fix:
```
pluggy.manager.PluginValidationError: Plugin 'pytest_cov' could not be loaded: (coverage 4.0.3 (/usr/local/lib/python2.7/site-packages), Requirement.parse('coverage>=4.4'))!
```

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,6 +44,7 @@ <h1>Index of python-pkgs</h1>
     <a href="contextlib2-0.6.0.post1-py2.py3-none-any.whl">contextlib2-0.6.0.post1-py2.py3-none-any.whl</a><br/>
     <a href="coverage-4.0.3-cp27-none-linux_x86_64.whl">coverage-4.0.3-cp27-none-linux_x86_64.whl</a><br/>
     <a href="coverage-4.0.3-cp27-none-macosx_10_10_x86_64.whl">coverage-4.0.3-cp27-none-macosx_10_10_x86_64.whl</a><br/>
+    <a href="coverage-4.4-cp27-cp27m-macosx_11_0_x86_64.whl">coverage-4.4-cp27-cp27m-macosx_11_0_x86_64.whl</a><br/>
```
Comment (MasterDDT) [inline index.html:48]: How come its using a diff version? We might want to pin 4.0.3 somewhere, and then you can make a mac 11 wheel for that

Comment (MasterDDT) [reply index.html:48]: I think here you can pin it https://github.com/ActionIQ/aiq/blob/42f116f8ff374cc75f3b4bdb970d47c2c4ff9407/build-support/versions/1.27.0/pants.ini#L285

In general with big sur, we should keep the library versions the same, you just need to be building new wheels for osx 11

Comment (oliviadawd) [reply index.html:48]: it's weird tho cos the error im getting says that it requires this package to be `>=4.4` which `4.0.3` is not :/

Comment (MasterDDT) [reply index.html:48]: What are you running when you get this failure? If its pants test, then ^^ line should be the right place.

If its initial pants initialization, then its pants-requirements.txt

If its some python build/etc, then its 3rdparty/python/requirements.txt

Comment (oliviadawd) [reply index.html:48]: i was trying to run a python test so i ran `aiq test test/python/journey_test`

Comment (oliviadawd) [reply index.html:48]: yea it seems like we need to upgrade this package cos when i pin the version in `pants.ini` i get this error when i run `aiq test test/python/journey_test` in the command line
```
Exception message: Failed to resolve compatible distributions:
1: pytest-cov==2.8.1 requires coverage>=4.4 but coverage 4.0.3 was resolved
```

Comment (MasterDDT) [reply index.html:48]: interesting, cc @LarryFinn 

Ok then can you pin coverage=4.4? I'll build a osx 10 wheel for it. And then this change looks fine

**Review by MasterDDT — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (oliviadawd) [reply index.html:48]: yep! i'll make a PR in the other repo 👍

**Pull request merged**